docs/docs/api/Tasks/fetchSingleTask.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Fetch Single Task
+---
+
+export const Endpoint = ({children, color}) => ( <span style={{
+borderRadius: '2px',
+color: '#E83E8C',
+}}>{children}</span> );
+
+<Endpoint>GET tasks/task/:id/</Endpoint>: Fetching a single Task
```

Comment:
Similar to the last comment on `createTasks.md` you should have the forward slash (`/`) before `tasks`

Reply:
Fixed: Changes are reflected in recent commits